From dd5748d456f7456c56453badbae3b063afbf96eb Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Thu, 14 May 2020 17:47:30 -0700
Subject: [PATCH] CMakeLists.txt: add missing tf to DEPENDS

* fixes costmap-cspace issue:
ros1-melodic-thud/tmp-glibc/work/core2-64-oe-linux/costmap-cspace/0.8.1-1-r0/recipe-sysroot-native/usr/bin/x86_64-oe-linux/../../libexec/x86_64-oe-linux/gcc/x86_64-oe-linux/8.2.0/ld: warning: libtf.so, needed by ros1-melodic-thud/tmp-glibc/work/core2-64-oe-linux/costmap-cspace/0.8.1-1-r0/recipe-sysroot/opt/ros/melodic/lib/liblaser_geometry.so, not found (try using -rpath or -rpath-link)
ros1-melodic-thud/tmp-glibc/work/core2-64-oe-linux/costmap-cspace/0.8.1-1-r0/recipe-sysroot-native/usr/bin/x86_64-oe-linux/../../libexec/x86_64-oe-linux/gcc/x86_64-oe-linux/8.2.0/ld: ros1-melodic-thud/tmp-glibc/work/core2-64-oe-linux/costmap-cspace/0.8.1-1-r0/recipe-sysroot/opt/ros/melodic/lib/liblaser_geometry.so: undefined reference to `tf::Transformer::lookupTransform(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::Time const&, tf::StampedTransform&) const'
collect2: error: ld returned 1 exit status

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 28ab530..629302a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@ catkin_python_setup()
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES laser_geometry
-  DEPENDS Boost Eigen
+  DEPENDS tf Boost Eigen
 )
 
 include_directories(include
